stacks/api/function/queue/node/database.ts

```diff
@@ -0,0 +1,61 @@
+import {Database} from "@spica-server/function/queue/proto";
+import * as grpc from "grpc";
+
+export class DatabaseQueue {
+  private client: any;
+
+  constructor() {
+    this.client = new Database.QueueClient("0.0.0.0:5678", grpc.credentials.createInsecure());
+  }
+
+  pop(e: Database.Change.Pop): Promise<Database.Change> {
+    return new Promise((resolve, reject) => {
+      this.client.pop(e, (error, event) => {
+        if (error) {
+          reject(new Error(error.details));
+        } else {
+          resolve(event);
+        }
+      });
+    });
+  }
+}
+
+export class Change<T = unknown> {
```

Comment:
@thesayyn What's the point of using unknown for a generic?

Reply:
https://stackoverflow.com/questions/51439843/unknown-vs-any/51439876

Reply:
Generic parameter is optional here.